core/templates/mathjaxConfig.ts

```diff
@@ -1,6 +1,10 @@
 window.MathJax = {
-  // Need this because technically the type error thrown here is correct.
-  // This is not the type of MathJax but MathJaxConfig.
+  // There is a typescript error here because window.MathJax should be a MathJax
```

Comment:
This throws "...". We suppress this error because the MathJax instructions at [LINK] explicitly instruct us to set window.MathJax to a MathJax config, which will only be changed to a MathJax object (which is what the typescript compiler expects) after the MathJax library loads.

Reply:
done